specs/eip4844/p2p-interface.md

```diff
@@ -81,11 +82,17 @@ The new topics along with the type of the `data` field of a gossipsub message ar
 
 EIP4844 introduces a new global topic for beacon block and blobs-sidecars.
 
+##### `beacon_block`
+
+This topic is deprecated and clients **MUST NOT** expose in their topic set to any peer. Implementers do not need to do
```

Comment:
Might be worth noting when to turn it off. 
The two safe options are either:
1. (easier) Turn off the topic one epoch after `4844_FORK_EPOCH` to allow for any weird late gossip
2. (harder) turn off after `4844_FORK_EPOCH` is finalized
    * This is a bit more "correct" wrt to the `beacon_block` p2p slot range validation -- "validate that `signed_beacon_block.message.slot > compute_start_slot_at_epoch(state.finalized_checkpoint.epoch)`"
    * But might induce undue complexity when option (1) is pretty damn fine

Reply:
the spec here: 

https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/p2p-interface.md#transitioning-the-gossip

suggests to me that nodes should wait two epochs from the fork and then unsubscribe this topic

do you think we need special consideration here?

Reply:
This is fine. I might make an explicit call out to that previous area in the spec as I think it has the potential to be missed